cmd/eno-reconciler/main.go

```diff
@@ -108,7 +110,7 @@ func run() error {
 	writeBuffer := flowcontrol.NewResourceSliceWriteBufferForManager(mgr, writeBatchInterval, 1)
 
 	rCache := reconstitution.NewCache(mgr.GetClient())
-	reconciler, err := reconciliation.New(mgr, rCache, writeBuffer, remoteConfig, discoveryMaxRPS, rediscoverWhenNotFound, readinessPollInterval)
+	reconciler, err := reconciliation.New(mgr, rCache, writeBuffer, remoteConfig, discoveryMaxRPS, rediscoverWhenNotFound, timeout, readinessPollInterval)
```

Comment:
Time to put these options in a reconciler specific struct or not yet? 👀

Reply:
Personally I think it's right on the threshold, like the next one definitely needs it.

Reply:
Thought about it more and pushed a commit to add one 🙃